nautobot/docs/user-guide/core-data-model/overview/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Understanding the data model is key to effectively navigating Nautobot. The number of models in Nautobot Core alone is nearly 200 models which does not include any of the models that come from Nautobot apps. This introduction does not attempt to document all of models, but instead intends to highlight the most common models to be used as a quick reference.
+
+## Primary Model Simplified
```

Comment:
Since `PrimaryModel` has a specific meaning within Nautobot, can we change this to something like `Basic Models` or `Fundamental Models` throughout this file?